src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Helper/ResultHelper.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using Microsoft.PowerToys.Run.Plugin.WindowsSettings.Classes;
+using Microsoft.PowerToys.Run.Plugin.WindowsSettings.Helper;
+using Wox.Plugin;
+using Wox.Plugin.Logger;
+
+namespace Microsoft.PowerToys.Run.Plugin.WindowsRegistry.Helper
+{
+    /// <summary>
+    /// Helper class to easier work with results
+    /// </summary>
+    internal static class ResultHelper
+    {
+        /// <summary>
+        /// Return a list with <see cref="Result"/>s, based on the given list
+        /// </summary>
+        /// <param name="list">The original result list to convert</param>
+        /// <param name="iconPath">The path to the icon of each entry</param>
+        /// <returns>A list with <see cref="Result"/></returns>
+        internal static List<Result> GetResultList(in IEnumerable<IWindowsSetting> list, in string iconPath)
+        {
+            var resultList = new List<Result>(list.Count());
+
+            foreach (var entry in list)
+            {
+                var subTitle = new StringBuilder();
+                subTitle.Append("Area: ");
+                subTitle.Append(SettingsHelper.GetAreaText(entry));
+
+                if (!string.IsNullOrEmpty(entry.Alt))
+                {
+                    subTitle.Append(" - alt. name: ");
```

Comment:
We should write whole words. They are better to translate. And we should have the same casing in all titles.


```suggestion
                    subTitle.Append(" - Alternative name: ");
```